trie/secure_trie.go

```diff
@@ -96,7 +100,7 @@ func (t *SecureTrie) TryUpdateAccount(key []byte, acc *types.StateAccount) error
 	if err != nil {
 		return err
 	}
-	if err := t.trie.TryUpdate(hk, data); err != nil {
+	if err := t.trie.tryUpdate(hk, data); err != nil {
```

Comment:
I feel like `TryUpdateAccount` and `TryGetAccount` are not aligned in SecureTrie.
Personally I think we should move all account encode/decode logic to Trie and SecureTrie is just a wrapper for hashing key.